src/cascadia/TerminalSettingsEditor/ViewModelHelpers.h

```diff
@@ -39,33 +39,33 @@ struct ViewModelHelper
     winrt::event<::winrt::Windows::UI::Xaml::Data::PropertyChangedEventHandler> _propertyChangedHandlers;
 };
 
-#define _BASE_OBSERVABLE_PROJECTED_SETTING(target, name)  \
-public:                                                   \
-    auto name() const noexcept { return target.name(); }; \
-    template<typename T>                                  \
-    void name(const T& value)                             \
-    {                                                     \
-        if (name() != value)                              \
-        {                                                 \
-            target.name(value);                           \
-            _NotifyChanges(L"Has" #name, L#name);         \
-        }                                                 \
-    }                                                     \
+#define _BASE_OBSERVABLE_PROJECTED_SETTING(target, name)                   \
+public:                                                                    \
+    auto name() const noexcept { return target.name(); };                  \
+    template<typename T>                                                   \
+    void name(const T& value)                                              \
+    {                                                                      \
+        if (name() != value)                                               \
+        {                                                                  \
+            target.name(value);                                            \
+            _NotifyChanges(L"Has" #name, L#name, L#name "OverrideSource"); \
```

Comment:
XxxOverrideSource *is* our parent. It changes (goes from returning null to non-null) if the value of HasXxx changes. Technically it doesn't matter if we notify on it because the listener is also listening for HasXxx changing.